system/Database/BaseBuilder.php

```diff
@@ -2744,7 +2762,7 @@ protected function isSubquery($value): bool
      * @param BaseBuilder|Closure $builder
      * @param bool                $wrapped Wrap the subquery in brackets
      */
-    protected function buildSubquery($builder, bool $wrapped = false): string
+    protected function buildSubquery($builder, bool $wrapped = false, string $alias = ''): string
```

Comment:
Please update docblock for this method.

Reply:
The rector removes ``@param $alias`` and ``@return`` attributes from the docblock.
If the behavior of the rector with the ``@param $alias`` attribute can be corrected by adding a description, then with the ``@return`` attribute I do not know what to do.

Reply:
Oh, I see. Let's try calling @samsonasik to the rescue on this one.

Reply:
@iRedds when there is no description, it is ok to remove the docblock, so current change is ok for me if it is already obvious 👍